wasm/validator/src/recursive.rs

```diff
@@ -0,0 +1,290 @@
+//! Module with structs and functions for recursive validation.
```

Comment:
Why this module is not part of the validator itself?

Reply:
I removed the module after the last changes